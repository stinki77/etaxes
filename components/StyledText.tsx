<<<<<<< HEAD
import { Text, TextProps } from './Themed';
=======
ï»¿import { Text, TextProps } from './Themed';
>>>>>>> restore/all

export function MonoText(props: TextProps) {
  return <Text {...props} style={[props.style, { fontFamily: 'SpaceMono' }]} />;
}
