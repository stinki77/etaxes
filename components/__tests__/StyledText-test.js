<<<<<<< HEAD
import * as React from 'react';
import renderer from 'react-test-renderer';

import { MonoText } from '../StyledText';

it(`renders correctly`, () => {
  const tree = renderer.create(<MonoText>Snapshot test!</MonoText>).toJSON();

  expect(tree).toMatchSnapshot();
=======
ï»¿import React from "react";
import { render } from "@testing-library/react-native";
import { MonoText } from "../StyledText";

describe("MonoText", () => {
  it("renders correctly", () => {
    const { getByText, toJSON } = render(<MonoText>Snapshot test!</MonoText>);
    expect(getByText("Snapshot test!")).toBeTruthy();
    expect(toJSON()).toMatchSnapshot();
  });
>>>>>>> restore/all
});
