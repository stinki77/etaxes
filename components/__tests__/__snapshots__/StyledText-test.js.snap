// Jest Snapshot v1, https://goo.gl/fbAQLP

<<<<<<< HEAD
exports[`renders correctly 1`] = `
=======
exports[`MonoText renders correctly 1`] = `
>>>>>>> restore/all
<Text
  style={
    [
      {
        "color": "#000",
      },
      [
        undefined,
        {
          "fontFamily": "SpaceMono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
